test/OffsetHelper.test.ts

```diff
@@ -58,15 +64,11 @@ describe("OffsetHelper", function () {
       ]
     );
 
+    bct = IToucanPoolToken__factory.connect(addresses.bct, addr2);
```

Comment:
nit: since these symbols are defined within the same function scope now, we could declare them as `const` here as we assign them, instead of declaring them first as `let` and then assigning them here. Does that make sense? (It would apply to all of the symbols within this fixture function)

Reply:
So this is something you actually have done in a future commit. So this is okay.